notifications.bs

```diff
@@ -317,6 +317,12 @@ meaning "<code>granted</code>". In that case
 "<code>granted</code>" is simply returned as there would be no reason
 for the application to ask for <a>permission</a>.
 
+<h3 id="permissions-integration">Permissions integration</h3>
+
+<p>The Notifications API is a <a>powerful feature</a>. [[!Permissions]]
```

Comment:
I saw some of the back-and-forth about this sentence. Isn't a problem here that #93 is still open?

Reply:
I guess there's 2 ways to go about this: 

1) land as-is (w/o "allowed in non-secure contexts" flag set), and do the work to close #93.
2) land with the "allowed in non-secure contexts" flag set, do the work to close #93, and when remove the flag at the same time.

I'm not sure I see any real advantage to either approach (but we should do the work to close #93, yes), the spec today is already not matching what WebKit, Gecko, and Chromium do wrt secure context requirements.

WDYT @annevk?

Reply:
Suggested we drop the flag, per 1.